### DISCUSSION TITLE
Add 3070 Ti

### DISCUSSION THREAD

**Discussion by derekhansell** [jef/streetmerchant#2617] [Ideas and feature requests]
Anyone have this working already?

**Comment by trillox9** [comment]
Streetmerchant is dead.... can't keep up with other python scraper bot. Puppeeteer is slow and not suitable to check a lot of element.
I was excited for the switching to cheerio, but i was noticed it was canceled, rip.

**Comment by derekhansell** [comment]
@trillox9 - thanks for the heads up. Are any of the python bots on github?

**Reply by trillox9** [reply]
Unfortunately not ready to use like streetmerchant, I haven't found one.
